crates/uv/src/cli.rs

```diff
@@ -1634,6 +1638,10 @@ pub(crate) struct RemoveArgs {
     #[arg(required = true)]
     pub(crate) requirements: Vec<PackageName>,
 
+    /// Remove the requirements from development dependencies.
+    #[arg(long)]
+    pub(crate) dev: bool,
+
```

Comment:
If I try to remove a dependency that's not present in the production dependencies but in the development dependencies should we just infer that you meant dev? Inference might be too far, but we should at least provide a hint suggesting `--dev` is what you wanted.

Reply:
See https://github.com/astral-sh/uv/pull/4327/files#diff-ba8cc7de2d3484ebe70b612b0ecb1ec65d09e426593158e93382cfc0af426640R47.

Reply:
Wow you're on it :)

Reply:
I guess I should actually read the pull request before I comment on the UX haha